set.mm

```diff
@@ -630953,6 +630953,1026 @@ standardize vectors to a length (norm) of one, but such definitions require
+      eqtrd mulcld adddird addcld addassd c3 cexp c2 cmul caddc c1 cmin df-3 cr
+      cc0 3re cn0 expp1d sqvald sqcld cq qre mulid2d cc sqmuld eqeltrd eqeltrrd
+      2nn0 subidd addid1d eqtr2d addsubassd addcomd 2timesd sq1 mulid1d binom2d
+      1cnd remulcld wi peano2re mpd resubcld wa 3nn nnq ax-mp qmulcl 3cubeslem1
+      cn jctil gt0ne0d wne 3ne0 mulne0d eqnetrd neneqd ) AUAZWMUBZDZBUCZWNDZUDZ
+      DZWMWPWNDZBWRDZUEZDZWMXBDZUJZAXDWMBWRDZUFZXBDZWPWNDZXFUGZDZWMWRDZXEAXDWTW
+      QWRDZXFXBDZXGXBDZWMWRDZXLAXDXNWMWRDZXGWMWRDZXBDZXPAXDXQWMXBDZXSAXDXMWMWRD
+      ZXFWMWRDZXBDZWMXBDZXTAXDYAWMBWMWRDZWRDZXBDZWMXBDZYDAXDYAWMXFWRDZXBDZWMXBD
+      ZYHAXDYAWMWMWRDZBWRDZXBDZWMXBDZYKAXDWTWQWMWRDZWRDZYMXBDZWMXBDZYOAXDWTWMWQ
+      WRDZWRDZYMXBDZWMXBDZYSAXDWTWMWRDZWQWRDZYMXBDZWMXBDZUUCAXDUUEXAXBDZWMXBDZU
+      UGAXDWMWPXGXBDZWNDZWQWRDZXAXBDZWMXBDUUIAXCUUMWMXBAWSUULXAXBAWOUUKWQWRAWMU
+      UJWMWNWMUUJEAUHFGHHHAUUMUUHWMXBAUULUUEXAXBAUUKUUDWQWRAWMWPAWMWMUIZIAUKFZJ
+      ZWPULIAVCFUMHHHPAUUHUUFWMXBAXAYMUUEXBAWTYLBWRAWMUUPUNHGHPAUUFUUBWMXBAUUEU
+      UAYMXBAWTWMWQAWMUUPUOZUUPAWQABABUPZIZBUUNICBUQKZLJZMHHPAYRUUBWMXBAYQUUAYM
+      XBAYPYTWTWRAWQWMUVAUUPNGHHOAYNYRWMXBAYAYQYMXBAWTWQWMUUQUVAUUPMHHOAYNYJWMX
+      BAYMYIYAXBAWMWMBUUPUUPABUUTJZMGHPAYGYJWMXBAYFYIYAXBAYEXFWMWRABWMUVBUUPNGG
+      HOAYCYGWMXBAYBYFYAXBAWMBWMUUPUVBUUPMGHOAXQYCWMXBAXMXFWMAWTWQUUQUVAQZAWMBU
+      UPUVBQZUUPRHOAXRWMXQXBAWMUUPURGOAXNXGWMAXFWPWNDZXFXBDZXNUSZAUVEXMXFXBAWMB
+      UUPUVBUTZHZAUVEXFAUVEXMUVGUVHUVCVAZUVDSZVBZAVMZUUPROAXOXKWMWRAXOUVEWPXFXG
+      WRDZWRDZXBDZXGWPWNDZXBDZXFXJDZXKAXOUVEWPXFWRDZXBDZUVQXBDZXFXJDZUVSAXOUWAX
+      GXBDZXFXJDZUWCAXOUVEXFXFXBDZXBDZXGXBDZXFXJDZUWEAXOUVFXFXBDZXGXBDZXFXJDZUW
+      IAXOUVFXHXBDZXFXJDZUWLAXOUVFXGXFXBDZXBDZXFXJDZUWNAXOUVFXGXBDZXFXBDZXFXJDZ
+      UWQAXOXOXFXBDZXFXJDZUWTAXOXOXFXFXJDZXBDZUXBAUXDXOXEXBDXOAUXCXEXOXBAXFUVDV
+      DGAXOAXNXGUVLUVMSZVEVFAXOXFXFUXEUVDUVDVGOAUWSUXAXFXJAUWRXOXFXBAUVFXNXGXBU
+      VIHHHOAUWSUWPXFXJAUVFXGXFUVKUVMUVDTHPAUWMUWPXFXJAXHUWOUVFXBAXFXGUVDUVMVHG
+      HOAUWKUWMXFXJAUVFXFXGUVKUVDUVMTHOAUWKUWHXFXJAUWJUWGXGXBAUVEXFXFUVJUVDUVDT
+      HHPAUWDUWHXFXJAUWAUWGXGXBAUVTUWFUVEXBAXFUVDVIGHHOAUWBUWDXFXJAUVQXGUWAXBUV
+      QXGEAVJFGHOAUVRUWBXFXJAUVPUWAUVQXBAUVOUVTUVEXBAUVNXFWPWRAXFUVDVKGGHHOAXIU
+      VRXFXJAXFXGUVDUVMVLHOHPAXKWMAXKAXIXFAXHAXFUUNIZXHUUNIZAWMBUUOUUTVNZUXFUXG
+      VOAXFVPFVQLUXHVRJUUPAXKAXFAWMUURIZUUSVSXFUURIAUUSUXICWMWEIUXIVTWMWAWBWFWM
+      BWCKWDWGWMXEWHAWIFWJWKWL $.
+
+    $( A polynomial identity used in the proof of ~ 3cubes .  (Contributed by
+       Igor Ieskov, 22-Jan-2024.)  (The assertion statement was formalized by
+       Saveliy Skresanov.) $)
+    3cubeslem3 $p |- ( ph ->
+      ( A x. ( ( ( ( ( 3 ^ 3 ) x. ( A ^ 2 ) )
+      + ( ( 3 ^ 2 ) x. A ) ) + 3 ) ^ 3 ) ) =
+      ( ( ( ( ( ( 3 ^ 3 ) x. ( A ^ 3 ) ) - 1 ) ^ 3 ) +
+      ( ( ( -u ( ( 3 ^ 3 ) x. ( A ^ 3 ) )
+      + ( ( 3 ^ 2 ) x. A ) ) + 1 ) ^ 3 ) ) +
+      ( ( ( ( 3 ^ 3 ) x. ( A ^ 2 ) ) + ( ( 3 ^ 2 ) x. A ) ) ^ 3 ) ) )
```

Comment:
Maybe the source code / ASCII statement would be a bit more readable if indented differently like for example:
```
     ( A x. ( ( ( ( ( 3 ^ 3 ) x. ( A ^ 2 ) )
            + ( ( 3 ^ 2 ) x. A ) ) + 3 ) ^ 3 ) ) 
  = ( ( ( ( ( ( 3 ^ 3 ) x. ( A ^ 3 ) ) - 1 ) ^ 3 )
      + ( ( ( -u ( ( 3 ^ 3 ) x. ( A ^ 3 ) )
      + ( ( 3 ^ 2 ) x. A ) ) + 1 ) ^ 3 ) )
      + ( ( ( ( 3 ^ 3 ) x. ( A ^ 2 ) ) + ( ( 3 ^ 2 ) x. A ) ) ^ 3 ) ) )
```
The idea is to have the equal sign `=` and the top level operations stand out a bit more.
As far as I know, our toolchain neither messes up with the formatting of formulas nor imposes any rules there.

Reply:
Thanks for the example. I reformatted all other long statements. The formatting for 3cubeslem4 takes quite a bit of space, but I didn't know how to make that long statement readable and use less space.